starter-core/src/main/java/io/micronaut/starter/feature/build/MicronautBuildPlugin.java

```diff
@@ -209,11 +223,19 @@ protected MicronautApplicationGradlePlugin.Builder micronautGradleApplicationPlu
         if (generatorContext.getFeatures().contains(AwsLambda.FEATURE_NAME_AWS_LAMBDA) && (
                 (generatorContext.getApplicationType() == ApplicationType.FUNCTION && generatorContext.getFeatures().contains(FEATURE_NAME_GRAALVM)) ||
                         (generatorContext.getApplicationType() == ApplicationType.DEFAULT))) {
-            builder = builder.dockerNative(Dockerfile.builder().baseImage("amazonlinux:2")
+            builder.dockerNative(Dockerfile.builder()
+                    .baseImage("amazonlinux:2")
```

Comment:
The default is 2023 in Gradle plugin 4.3.0, see https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/929

Reply:
This is 4.2.x

Reply:
Looks like I need to rebase to 4.3.x